spec.js

```diff
@@ -0,0 +1,61 @@
+describe('Smoke test for demo app', function() {
```

Comment:
you should format test suite title as "Test suite id: Test suite title" from your test case document